### DISCUSSION TITLE
compilation faild on centos7

### DISCUSSION THREAD

**Discussion by buzhaodi** [alibaba/xquic#69] [Q&A] [answered]
I compiled it under CentOS 7. Boringssl has been installed，The GCC version is 5.2.1 The cmkae version is 3.2.0 This error occurred during compilation.what can i do?
我再centos7下编译，已经安装了boringssl,gcc版本是5.2.1.cmkae版本是3.2.0.编译出现这个错误。
![image](https://user-images.githubusercontent.com/16113837/150713046-dae53141-d752-4ec1-a9ca-5050d2419b2b.png)

**Comment by ruiqizhou** [comment, marked as answer]
Did u follow the quickstart guide? https://github.com/alibaba/xquic#build-with-boringssl

you can check your SSL_PATH.

**Reply by buzhaodi** [reply]
Thank you. I've solved it according to your tips. The reason is that when I switch GCC to 5.2.1

scl enable devtoolset-4 bash

The command clears the environment variables。and it will be fine after resetting
